cmd/installer/main.go

```diff
@@ -82,6 +82,10 @@ func parseOptions() (installerConfig, error) {
 	if flags.skipLivenessProbes {
 		options.HealthzCheckPort = 0
 	}
+	if flags.apiServerReplicas != 1 {
```

Comment:
totally nit but this condition is not useful I think, you can always set the APIServerReplicas.